pkg/health/server/server.go

```diff
@@ -105,7 +105,7 @@ func (s *Server) DumpUptime() string {
 // getNodes fetches the latest set of nodes in the cluster from the Cilium
 // daemon, and updates the Server's 'nodes' map.
 func (s *Server) getNodes() (nodeMap, error) {
-	scopedLog := logrus.NewEntry(log)
+	scopedLog := log
```

Comment:
@raybejjani why the previous NewEntry?

Reply:
I didn't add it but it's similar to a logrus.WithFields but it precreates the fields map (looking at the code). I think it's safe to do what this PR does.